addons/mass_mailing/static/src/models/trace.js

```diff
@@ -0,0 +1,122 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core'
+import { attr } from '@mail/model/model_field'
+
+registerModel({
+    name: 'MailingTrace',
+    modelMethods: {
+        /**
+        * @param {Object} data
+        * @return {Object}
+        */
+         convertData(data) {
```

Comment:
new models should not introduce convert data, have the caller directly return the proper fields instead (it looks like it is already the case)